app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -66,6 +69,11 @@ interface ICloneRepositoryState {
   readonly shouldClearFilter: boolean
 }
 
+interface ICloneData {
```

Comment:
I'm not a huge fan of "data" as a name - can we think of something better here to represent what this is? `ICloneDetails`? Something else?